mastering-plone/reusable.rst

```diff
@@ -165,8 +165,8 @@ Lets move on to :file:`browser/viewlets.py`
 
     class Vote(base.ViewletBase):
 
-         ...
-         can_vote = None
+        ...
```

Comment:
To make this valid Python code, and to have proper syntax highlighting with Pygments, you need to make non-code stuff into comment.

```python
# ...
```

Please change all the above instances accordingly.

Also check that the highlighted (emphasized) lines still align with the intent.

Reply:
I did it, only on python code. I am not sure if it would be good to comment zcml, page templates or xml files.

Reply:
If you detect problems either with syntax colorization or highlighting, then it's a good idea to fix the syntax of the code to comply with the language option in the `code-block`.

In another project, I ran into some issues with recent versions of Sphinx where syntax is becoming more strictly enforced. That is a good thing because it helps me catch mistakes.